src/Java.Interop/Java.Interop/JniRuntime.cs

```diff
@@ -60,6 +60,7 @@ public partial class CreationOptions {
 
 			public  JniObjectReferenceManager   ObjectReferenceManager      {get; set;}
 			public  JniTypeManager              TypeManager                 {get; set;}
+			public  string                      JvmDllPath                  {get; set;}
```

Comment:
Sorry, but `JvmDllPath` looks ugly to me. :-(

Perhaps `JvmLibraryPath`?

Reply:
Really? The `jvm.dll` is used in existing *dllmap* entry, so I just reused that.